## VSC0030: Invalid target property

| Property         | Value                     | 
| ---------------- | ------------------------- | 
| ID               | VSC0030                   | 
| Category         | Compiler                  | 
| Severity         | Error                     | 
| Runtime Version  | 2021.11.24.16.19 ~ latest | 
| Compiler Version | 0.20.3 ~ latest           | 

Invalid target property\.  

### Example

#### Code with Diagnostic


```csharp
using UdonSharp;

class TestBehaviour : UdonSharpBehaviour
{
    [FieldChangeCallback("MissingProperty")]
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    public string _foo;
    ~~~~~~~~~~~~~~~~~~~
}
```

#### Code with Fix


```csharp
// NOT YET PROVIDED
```


